app/controllers/admin/custom_emojis_controller.rb

```diff
@@ -2,19 +2,22 @@
 
 module Admin
   class CustomEmojisController < BaseController
-    before_action :set_custom_emoji, except: [:index, :new, :create]
-    before_action :set_filter_params
-
     include ObfuscateFilename
+
+    before_action :set_custom_emoji, except: [:index, :new, :create, :batch]
```

Comment:
Is there an action for which it is used? If so, wouldn't `only` be more fitting than `except`?